CONTRIBUTING.md

```diff
@@ -30,11 +30,32 @@ For Git commit messages, our principle is that `git log --pretty=oneline` should
 - Does your PR include appropriate tests?
 - Make sure your PR title and description makes it easy for other developers to understand what the contained commits do. The title should say what the changes do. The description should expand on what it does (if not obvious from the title alone), and say why it is being done.
 - If your PR corresponds to an issue, add “Fixes #XX” to your pull request description. This will auto-close the corresponding issue when the commit is merged into master and tie the PR to the issue.
-- If your PR includes user-facing changes, you must add a line describing the change to [unreleased.rst](unreleased.rst) as part of your PR. Each entry in this document must start with the component to which is belongs, as in the following example:
+- If your PR includes user-facing changes, one or more commits in the PR must include in their body with a section between the ``CHANGELOG_BEGIN`` and ``CHANGELOG_END`` tags that includes relevant changelog entries, where each entry starts with the component to which is belongs in square brackets. Use RST to format links as this text will be added to the changelog upon release.
+
+The following is an example of a well-formed commit, including the description (first line) and a body that includes changelog additions:
+
+      Fixes #1311
+
+      Also fixes a typo in the Scala bindings documentation.
+
+      CHANGELOG_BEGIN
 
       - [Sandbox] Introduced a new API for package management.
         See `#1311 <https://github.com/digital-asset/daml/issues/1311>`__.
 
+      CHANGELOG_END
+
+If you want to amend an existing changelog entry part of a PR already merged on master, do so with the following format:
+
+      CHANGELOG_BEGIN
+
+      ** Replace changelog for PR #42 **
```

Comment:
Not sure how useful the PR number is. Release notes usually either mention issues or neither PRs or issues so how do I figure out which line I need to change? tbf, this is not super important given that I hope this will be very rare.

Reply:
I agree, let's see how this evolves, we can change it over time. Ironically. :wink: